packages/dd-trace/src/appsec/reporter.js

```diff
@@ -75,15 +81,27 @@ const requestHeadersList = [
   ...identificationHeaders
 ]
 
+const authenticationHeadersList = [
```

Comment:
could call it "redactedHeadersList" instead, so we know what it will be used for directly ?